src/test/util_tests.cpp

```diff
@@ -600,20 +601,22 @@ BOOST_AUTO_TEST_CASE(util_GetChainName)
 //   outside a network section, and non-network specific settings like "-server"
 //   that aren't sensitive to the network.
 //
-struct SettingsMergeTestingSetup : public BasicTestingSetup {
+struct ArgsMergeTestingSetup : public BasicTestingSetup {
     //! Max number of actions to sequence together. Can decrease this when
     //! debugging to make test results easier to understand.
     static constexpr int MAX_ACTIONS = 3;
 
-    enum Action { SET = 0, NEGATE, SECTION_SET, SECTION_NEGATE, END };
+    enum Action { END, SET, NEGATE, SECTION_SET, SECTION_NEGATE };
```

Comment:
In commit 00093ed7aae76d98792cfe9e3706ec3e27be2a13

Shouldn't this be named (or similar)
```suggestion
    enum Action { NONE, SET, NEGATE, SECTION_SET, SECTION_NEGATE };
```